CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,122 @@
+
+## Nos normes
+
+Exemples de comportements qui contribuent à créer un environnement positif incluent :
+
+* Utiliser un langage accueillant et inclusif
+* Être respectueux des différents points de vue et expériences
+* Accepter gracieusement les critiques constructives
+* Se concentrer sur ce qui est le mieux pour la communauté
+* Faire preuve d'empathie envers les autres membres de la communauté
+
+Voici des exemples de comportements inacceptables de la part des participants :
+
+* L'utilisation d'un langage ou d'images sexualisés et d'une attention sexuelle importunée, ou percées
+* Trollage, commentaires insultants ou méprisants, et attaques personnelles ou politiques
+* Harcèlement public ou privé
+* La publication d'informations privées d'autrui, telles que des informations physiques ou électroniques. adresse, sans autorisation explicite
+* Tout autre comportement qui pourrait raisonnablement être considéré comme inapproprié dans le cadre d'une enquête du contexte professionnel
+
+## Nos responsabilités
+
+Les responsables de la mise à jour du projet ont la responsabilité de clarifier les normes d'acceptabilité du et on s'attend à ce qu'ils prennent des mesures correctives appropriées et équitables en cas de comportement inacceptable.
+
+Les responsables de projet ont le droit et la responsabilité de supprimer, d'éditer ou de rejeter les commentaires, les soumissions (commits), le code, les éditions du wiki, les problèmes et autres contributions qui ne sont pas conformes au présent Code de conduite, ou d'interdire temporairement ou définitivement tout contributeur pour d'autres comportements qu'ils jugent inappropriés, menaçant, offensant ou nuisible.
+
+## Portée
+
+Ce Code de conduite s'applique dans tous les espaces du projet, et il s'applique également lorsque une personne représente le projet, sa communauté dans les espaces publics ou [`Emploi et développement social Canada`].
+Des exemples de représentation d'un projet, d'une collectivité ou [`Emploi et développement social Canada`] comprennent l'utilisation d'un représentant officiel de la l'adresse électronique du projet, l'affichage par l'entremise d'un compte officiel de médias sociaux ou le fait d'agir à titre intérimaire en tant que représentant désigné lors d'un événement en ligne ou hors ligne.
+La représentation d'un projet peut être mieux défini et clarifié par les responsables du projet.
+
+## Application des règles
+
+Les cas de comportement abusif, de harcèlement ou d'autres comportements inacceptables peuvent être rapportés en communiquant avec l'équipe de projet à l'adresse suivante :[`ESDC.SD.DEV-DEV.DS.EDSC@servicecanada.gc.ca`].
```

Comment:
```suggestion
Les cas de comportement abusif, de harcèlement ou d'autres comportements inacceptables peuvent être rapportés en communiquant avec l'équipe de projet à l'adresse suivante&nbsp;: [`ESDC.SD.DEV-DEV.DS.EDSC@servicecanada.gc.ca`].
```

Reply:
I switched this text to point to GCpedia instead of email.